.github/workflows/deploy-cloudrun-credentials-it.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         npm run build
 
     - name: Set up authentication
-      uses: google-github-actions/setup-gcloud@master
```

Comment:
Should we leave this as is? Its good way to make sure own own workflows pass with latest setup-gcloud.

Reply:
Done